core-services/pdc/src/main/java/org/silverpeas/core/pdc/pdc/importexport/AxisType.java

```diff
@@ -29,19 +29,29 @@
  */
 package org.silverpeas.core.pdc.pdc.importexport;
 
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
 import java.util.List;
 
-/**
- * Classe utilisee pour le mapping castor
- * @author sdevolder
- */
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.NONE)
 public class AxisType {
 
+  @XmlAttribute
   private int id;
+  @XmlAttribute(name = "pdcValuePath")
   private String path;
+  @XmlElement(name = "pdcValue")
   private List<PdcValueType> listPdcValueType;
+  @XmlElement(name = "name")
   private String name;
 
+  public AxisType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)